doc/user/content/sql/system-catalog/mz_internal.md

```diff
@@ -369,19 +381,35 @@ Field       | Type       | Meaning
 `worker_id` | [`bigint`] | The ID of the worker thread hosting the dataflow.
 `time`      | [`mz_timestamp`] | The next timestamp at which the dataflow may change.
 
-### `mz_cluster_replica_metrics`
-
-The `mz_cluster_replica_metrics` table gives the last known CPU and RAM utilization statistics
-for all processes of all extant cluster replicas.
-
-At this time, we do not make any guarantees about the exactness or freshness of these numbers.
-
-Field         | Type       | Meaning
---------------|------------|--------
-`replica_id`  | [`bigint`] | The ID of a cluster replica.
-`process_id`  | [`bigint`] | An identifier of a compute process within a replica.
-`cpu_nano_cores`   | [`bigint`] | Approximate CPU usage, in billionths of a vCPU core.
-`memory_bytes`| [`bigint`] | Approximate RAM usage, in bytes.
+### `mz_source_status`
+
+The `mz_source_status` view provides the current state for each source in the
+system, including potential error messages and additional metadata helpful for
+debugging.
```

Comment:
Internally a few folks have been surprised that this status table doesn't include single-row errors like invalid formats or envelopes on a message. (Those already show up in the same way errors in a materialized view do: when you query the collection.) I think it makes sense to handle these differently -- it's useful to know that your Kafka source is running, even if a single record had some invalid data in it -- but I worry it might be confusing to users. Maybe there's some way to mention the distinction here? (Whole-source errors vs. issues affecting a single row? Errors that prevent the source from making progress? I'm not sure what's the right way to communicate it.)

Reply:
> it's useful to know that your Kafka source is running, even if a single record had some invalid data in it

We don't have a way to handle single-row errors that isn't dropping and recreating the source, though...right? I'll merge it as is, and we can adjust later on if it turns out to be confusing (mostly because I also can't find the words to express this nuance right now! 😅 ).